tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/types/TupleTypeBalaTest.java

```diff
@@ -85,6 +93,40 @@ public void testTupleTypeNegative() {
         Assert.assertEquals(negativeResult.getErrorCount(), i);
     }
 
+    @Test
```

Comment:
These should probably go in the relevant annotation tests. Can you check and move?